src/java/org/apache/cassandra/transport/messages/QueryMessage.java

```diff
@@ -113,6 +114,13 @@ protected boolean isTrackable()
 
             QueryHandler queryHandler = ClientState.getCQLQueryHandler();
             statement = queryHandler.parse(query, state, options);
+            if (statement instanceof AlterSchemaStatement){
+                logger.info(String.format("DDL Query: %s, Keyspace: %s, by User: %s, From: %s",
+                                          query,
+                                          ((AlterSchemaStatement) statement).keyspace(),
+                                          state.getClientState().getUser().getName(),
```

Comment:
using the getUser().getName here returns the actual user name, returns "anonymous" with auth is not enabled.